elements/rh-scheme-toggle/rh-scheme-toggle.css

```
@@ -0,0 +1,96 @@
+fieldset {
+  align-items: center;
+  border: 0;
+  display: flex;
+  flex-flow: row nowrap;
+  gap: var(--rh-space-lg);
+  margin-block-end: var(--rh-space-4xl);
+  padding: 0;
+
+  legend {
+    float: inline-start;
+  }
+
+  #button-group {
+    --rh-icon-size: var(--rh-size-icon-01);
+
+    display: flex;
+  }
+}
+
+label {
+  align-items: center;
+  background-color: initial;
+  border: var(--rh-border-width-sm) solid var(--rh-color-border-subtle);
```

```suggestion
  border: var(--rh-border-width-sm, 1px) solid var(--rh-color-border-subtle);
```

`--rh-color-border-subtle` already takes care of `light-dark` stuff, so it doesn't need a fallback here.

I find it helpful to reference the [variable definitions JSON](https://cdn.jsdelivr.net/npm/@rhds/tokens@3.0.0/json/rhds.tokens.flat.json) when looking at this stuff.